cryptofeed/exchange/bitmax.py

```diff
@@ -0,0 +1,93 @@
+'''
+Copyright (C) 2017-2019  Bryant Moscon - bmoscon@gmail.com
+
+Please see the LICENSE file for the terms and conditions
+associated with this software.
+'''
+import json
+import logging
+from decimal import Decimal
+
+from sortedcontainers import SortedDict as sd
+
+from cryptofeed.feed import Feed
+from cryptofeed.defines import BITMAX
+from cryptofeed.defines import TRADES, BUY, SELL, BID, ASK, L2_BOOK
+from cryptofeed.standards import pair_exchange_to_std, timestamp_normalize, pair_std_to_exchange
+
+
+LOG = logging.getLogger('feedhandler')
+
+
+class Bitmax(Feed):
+    id = BITMAX
+
+    def __init__(self, pairs=None, channels=None, callbacks=None, config=None, **kwargs):
+        if pairs is None:
+            pairs = list(set([pair for s in config.values() for pair in s]))
+        if len(pairs) != 1:
+            LOG.error("Bitmax requires a websocket per trading pair")
```

Comment:
There are ways around this - it was previously done for gemini, see https://github.com/bmoscon/cryptofeed/blob/ef4b6621efa974be829380d22323e2c71696a7c8/cryptofeed/feedhandler.py#L216

Reply:
thanks! I reintroduced the multi-pair method as you used with Gemini before, so it should be good now. 

If another exchange/feed comes along with the same requirements, i'll look into abstracting it away somehow.